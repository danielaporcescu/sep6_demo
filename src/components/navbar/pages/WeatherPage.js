import React from "react";
import "../../../App.css";
import WeatherPage from "../../pages/weather-page";

export default function Weather() {
  return (
    <>
      <div className="weather-page">
        <WeatherPage />
      </div>
    </>
  );
}
