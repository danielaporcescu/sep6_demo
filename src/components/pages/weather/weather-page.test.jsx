import React from 'react';

import Enzyme, { shallow, render, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import WeatherPage from './/weather-page';

Enzyme.configure({ adapter: new Adapter() })

it('Renders correctly FlightsPage', () => {
    const wrapper = shallow(<WeatherPage />)
  
    expect(toJson(wrapper)).toMatchSnapshot();
  });