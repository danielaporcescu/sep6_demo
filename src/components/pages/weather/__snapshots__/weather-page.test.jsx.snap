// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly FlightsPage 1`] = `
<div
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(Grid))
    container={true}
    justify="space-around"
    spacing={2}
  >
    <WithStyles(ForwardRef(Grid))
      item={true}
      md={6}
    >
      <WithStyles(ForwardRef(Paper))
        className="makeStyles-paper-2"
      >
        <Styled(MuiBox)
          paddingBottom={2}
        >
          How many weather observations there are for the origins in a table
        </Styled(MuiBox)>
        <Loader />
      </WithStyles(ForwardRef(Paper))>
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      md={6}
    >
      <WithStyles(ForwardRef(Paper))
        className="makeStyles-paper-2"
      >
        <Styled(MuiBox)
          paddingBottom={2}
        >
          The daily mean temperature (in Celsius) at JFK
        </Styled(MuiBox)>
        <Loader />
      </WithStyles(ForwardRef(Paper))>
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      md={12}
    >
      <WithStyles(ForwardRef(Paper))
        className="makeStyles-paper-2"
      >
        <Styled(MuiBox)
          paddingBottom={2}
        >
          For each of the three origins, all temperature attributes
        </Styled(MuiBox)>
        <Loader />
      </WithStyles(ForwardRef(Paper))>
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      md={12}
    >
      <WithStyles(ForwardRef(Paper))
        className="makeStyles-paper-2"
      >
        <Styled(MuiBox)
          paddingBottom={2}
        >
          The temperature (in Celsius) at JFK.
        </Styled(MuiBox)>
        <Loader />
      </WithStyles(ForwardRef(Paper))>
    </WithStyles(ForwardRef(Grid))>
    <WithStyles(ForwardRef(Grid))
      item={true}
      md={12}
    >
      <WithStyles(ForwardRef(Paper))
        className="makeStyles-paper-2"
      >
        <Styled(MuiBox)
          paddingBottom={2}
        >
          The daily mean temperature (in Celsius) for each origin in the same plot
        </Styled(MuiBox)>
        <Loader />
      </WithStyles(ForwardRef(Paper))>
    </WithStyles(ForwardRef(Grid))>
  </WithStyles(ForwardRef(Grid))>
</div>
`;
